scripts/release_processes/refresh.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
ToL: Possibly akin to the "Don't use `Some.apply()`!" recommendations.

```suggestion
#!/usr/bin/env bash
```

https://unix.stackexchange.com/questions/206350/what-is-the-difference-if-i-start-bash-with-bin-bash-or-usr-bin-env-bash